client/src/features/signup/components/LoginInfoForm/LoginInfoForm.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Field } from 'formik';
+import { TextField } from 'formik-mui';
+import { Grid, Typography } from '@mui/material';
+import { formModel } from '../../data';
+
+const LoginInfoForm = () => {
+  const { email, login, password, retypePassword } = formModel;
+
+  return (
+    <>
+      <Typography variant="h3" mb={3}>
+        Login Info
+      </Typography>
+      <Grid container spacing={3} mb={3}>
+        <Grid item xs={12} laptop={6}>
+          <Field component={TextField} name={email.name} label={email.label} fullWidth required />
+        </Grid>
+
+        <Grid item xs={12} laptop={6}>
+          <Field component={TextField} name={login.name} label={login.label} fullWidth required />
+        </Grid>
+
+        <Grid item xs={12} laptop={6}>
+          <Field component={TextField} name={password.name} label={password.label} fullWidth required />
```

Comment:
I think that it`s better to hide the password here.
![image](https://user-images.githubusercontent.com/112622259/202896787-1cc80165-a390-446c-b5eb-53bf2bd4af0c.png)

Reply:
agree, this will be performed in a future update including the mui InputAdornment component